CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.10.0rc1
+Features:
+-
+
+Bug Fixes:
+- Fixed bug where unsafe packages would get pinned in generated requirements files
+when `--allow-unsafe` was not set. ([#508](https://github.com/jazzband/pip-tools/pull/508)).
```

Comment:
Unsafe packages were being pinned in requirements files even when `--allow-unsafe` wasn't set instead of being comments